horovod/_keras/__init__.py

```diff
@@ -182,7 +182,7 @@ def __filtered_reduce_grads(grads, vars):
                                 rg.append(grad)
 
                     rg = self._allreduce_grads(rg, rv)
-                    horovod_size = size_op(process_set_id=self.process_set.process_set_id) if int(os.environ.get("HOROVOD_ELASTIC", 0)) else self.process_set.size()
+                    horovod_size = tf.cast(size_op(process_set_id=self.process_set.process_set_id), tf.float32) if int(os.environ.get("HOROVOD_ELASTIC", 0)) else self.process_set.size()
```

Comment:
Having a size of type `float` is a bit surprising, given the else branch is still an int. I'd rather have `float32(horovod_size)` wherever division by `horovod_size` occurs, or introduce a variable `horovod_size_float = float32(horovod_size)` and use that.

Reply:
@EnricoMi Agreed. I just updated to float(horovod_size) wherever division by horovod_size occurs. Thanks!